canvas.gypi

```diff
@@ -0,0 +1,43 @@
+{
+    'variables':
+    {
+        'has_FBDev%': 'true',
```

Comment:
Actually, there are a lot of cases where it's not available. Many headless server kernels are built without it, including most (or all?) EC2 and GCP images.